packages/test/test-gc-sweep-tests/src/dataObjectWithChildDataObject.ts

```diff
@@ -0,0 +1,71 @@
+/*!
```

Comment:
Consider moving all data stores into a single file `testDataObjects.ts` so that all different data object implementations are encapsulated in one place. It will decrease overhead and make it easier to navigate the code.

Reply:
This will be a later consideration, separate files allow for separate changes with fewer merge conflicts, we can move it together at the end